styles/style.css

```diff
@@ -117,10 +128,17 @@ button {
   font-weight: 700;
```

Comment:
Could also add a `margin-right` property here to add some space between the pictures and the writing